components/lsp/MintForm.tsx

```diff
@@ -140,7 +145,7 @@ const MintForm: FC<Props> = ({
           );
 
           // update our known allowance once approval passes
-          return approveTx.wait(1).then(() => getKnownAllowance());
+          return approveTx.wait(CONFIRMATIONS).then(() => getKnownAllowance());
```

Comment:
I think this explains why Tom needed to hard refresh the page for updating the balances, but I couldn't reproduce it